platform/testing/end-to-end/cypress-best-practices-on-vsp.md

```diff
@@ -22,6 +28,48 @@ End-to-end (e2e) testing on VA.gov is accomplished using a frontend testing fram
 
 The following documentation details Cypress best practices on VSP.
 
+## Running tests <a name="running-tests"></a>
+
+Cypress supports Chrome, Edge, Firefox, and a few [others](https://docs.cypress.io/guides/guides/launching-browsers.html#Browsers). You can run tests in headless mode or via the test runner.
+
+### Headless mode <a name="headless-mode"></a>
+
+To run headless tests, run `yarn cy:run`.
+
+By default, `yarn cy:run` runs Cypress tests headlessly in an Electron browser. You may specify another browser, and if you would like to run headless tests in another browser, you will have to explicitly include the `--headless` flag.
+
+```
+yarn cy:run --headless --browser firefox
+```
+
+You may experience some performance issues where particular long-running tests (such as an exhaustive test of a form) may take an extremely long time (unless your setup is optimized for them, as in the case of our CI, which tests the production build and also has the specs to cancel out the performance burden).
+
+For this reason, for local development, it might be better to run specific specs, even more so in the test runner. The Cypress team is [investigating various issues regarding performance](https://github.com/cypress-io/cypress/issues/6388#issuecomment-648795870) that may likely be related to this.
+
+### Test runner <a name="test-runner"></a>
+
+To run tests in the Cypress [test runner](https://docs.cypress.io/guides/core-concepts/test-runner.html#Overview), run `yarn cy:open`.
+
+There is a dropdown menu in the top right of the test runner that allows you to select a browser in which to run your tests. In our experience, Firefox has yielded the fastest test runs when testing locally, although it is currently a beta feature. The tests in CI will run in the default browser, which is Electron.
+
+The test runner provides the ability to pause tests, and [time travel](https://docs.cypress.io/guides/getting-started/writing-your-first-test.html#Time-travel), which allows you to see snapshots of your tests.
+
+With the test runner, you are able to use Cypress's "Selector Playground". This allows you to click on elements in the DOM and copy that element's selector to use in your test. However, [as mentioned earlier](#interacting-with-page-elements), selecting elements by CSS attributes is discouraged in favor of selecting by test id, which is in turn considered a fallback if selecting by other attributes (label text, role, etc.) is not feasible. In fact, the Selector Playground follows this best practice and automatically attempts to determine the selector by looking at `data-cy`, `data-test`, and `data-testid` before falling back to a CSS selector.
```

Comment:
The link "as mentioned earlier" doesn't appear to be pointing to a heading in the doc. Is there another one we could refer back to?

Reply:
Yep, reconciled this.